docs/semgrep-app/integrations.md

```diff
@@ -8,12 +8,104 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrations
 
+Semgrep App contains third party integrations to allow users to add data from Semgrep to other tools that are part of their workflows.
+
+Currently, Semgrep App integrates with the following tools:
+
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team |
+| Amazon S3 | Team |
+| Webhook | Team |
+
+## Where are integrations found?
+
+After logging into your Semgrep App account, you can find **Integrations** as a tab within **Settings.**
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
```

Comment:
```suggestion
![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)
```

Reply:
Maybe try `</p>` to achieve a similar result which would be less problematic in different browsers?

Reply:
@adamkvitek We'll fix this eventually, but keeping it for now to maintain layout consistency with the rest of the docs. I will also be resolving other similar issues. Thank you :)